### ISSUE TITLE
Menubar role and accessible name is not announced in Focus mode

### ISSUE THREAD

**Issue by joppekroon** [nvaccess/nvda#17448]
### Steps to reproduce:
1. Browse to the [APG menubar example](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-editor/)
2. TAB to the first menuitem in the example
3. Observe NVDA does not announce that a menubar has been entered, nor it's acccessible name.

### Actual behavior:

NVDA Speech: "Font  subMenu  1 of 4"

### Expected behavior:

NVDA Speech: "Text Formatting  menu bar Font subMenu 1 of 4"

#### NVDA version:
2024.4.1.34550

#### Windows version:
Microsoft Windows 10 Enterprise

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
